src/core/include/openvino/op/assign.hpp

```diff
@@ -31,6 +31,11 @@ class OPENVINO_API Assign : public util::AssignBase {
     std::shared_ptr<Node> clone_with_new_inputs(const OutputVector& new_args) const override;
 
     bool visit_attributes(AttributeVisitor& visitor) override;
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
Why do we need to have new evaluate methods?

Reply:
There was no evaluation for read_value and assign. But as @iefode commented above, I'll implement evaluate() in evaluate_map.cpp.